xml/upgrade-2.10.xsl

```diff
@@ -138,6 +138,15 @@
    Move ctxt:      N/A
    -->
   <cibtr:table for="resource-instance-attributes" msg-prefix="Resource instance_attributes">
+    <cibtr:replace what="pcmk_arg_map"
+                   with=""/>
+    <!-- simplified as pcmk_arg_map can encode multiple
+         comma-separated pairs (everything would be dropped then,
+         except for a single dangling case: "port" coming first) -->
+    <cibtr:replace what="pcmk_arg_map"
+                   with="pcmk_host_argument"
+                   in-case-of-droppable-prefix="port:"/>
+
```

Comment:
Do you plan to add regression tests for this one?

Reply:
My brain asynchronously told me I likely forgot to add the existing file.
And indeed, thanks for quick confirmation :-)
Other thing hit my consciousness was that `in-case-of-droppable-prefix`
should be guarded with a check that, in addition, comma is not included,
for good measure (said "dangling" case).